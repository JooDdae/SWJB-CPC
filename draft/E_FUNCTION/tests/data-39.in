71
4 12 63 69 45 53 13 17 42 7 58 16 14 46 20 9 36 50 3 32 66 34 43 41 49 47 54 57 28 59 60 56 62 19 64 22 35 68 55 -1
12 27 66 11 63 21 3 45 33 65 67 43 23 8 39 52 10 35 9 36 38 48 16 6 46 26 34 58 31 56 19 32 25 28 5 54 18 42 47 61 24 44 60 70 64 -1
36 61 29 52 47 51 32 14 55 15 58 66 33 54 64 22 42 23 48 45 39 8 4 57 41 10 25 38 9 21 59 35 31 44 18 7 37 17 5 12 67 46 6 49 11 28 40 27 56 -1
27 66 52 53 71 41 37 15 28 19 -1
65 61 45 -1
24 15 18 66 22 36 37 48 29 45 44 52 42 -1
27 64 63 70 10 47 36 8 66 51 55 21 12 2 7 19 67 68 35 29 9 42 5 54 40 48 49 46 39 16 43 4 17 69 34 60 58 32 11 22 13 31 20 44 28 65 41 71 38 53 30 24 26 52 6 50 62 57 18 -1
17 71 38 18 33 16 34 4 14 51 62 35 2 28 9 43 21 64 65 66 55 15 22 61 26 46 7 11 56 36 24 60 25 10 8 27 6 68 19 12 63 70 29 54 50 69 41 42 39 45 23 32 67 37 31 40 53 48 59 49 20 57 13 30 3 47 58 -1
54 20 23 46 37 6 26 51 38 35 2 62 57 52 66 29 33 18 53 61 28 12 64 11 63 22 36 27 42 32 68 40 50 65 43 48 31 13 25 39 70 59 34 45 9 5 67 8 4 60 16 -1
53 57 22 19 2 39 21 -1
50 69 39 55 25 4 11 33 63 61 5 16 8 53 9 51 65 15 26 70 41 18 57 49 66 60 62 48 7 44 6 36 2 46 28 40 38 21 58 34 68 71 13 19 22 45 27 12 43 29 23 52 59 14 67 42 32 31 24 37 10 35 47 20 54 17 -1
67 39 26 61 23 30 62 21 64 32 29 2 12 10 46 -1
12 -1
46 5 2 42 34 45 10 56 55 9 20 70 71 57 64 4 67 30 39 65 27 12 14 13 44 28 49 53 38 62 52 17 16 -1
14 12 60 48 15 10 21 49 2 27 67 13 61 70 69 16 47 66 68 35 58 45 37 39 6 62 56 26 25 5 34 51 55 63 29 19 59 18 57 22 53 33 71 46 4 28 9 40 11 32 7 41 50 -1
2 19 9 51 18 32 24 17 48 49 16 50 5 54 45 7 71 26 61 60 37 47 56 4 65 68 57 22 29 55 31 8 13 58 14 20 43 66 23 46 -1
50 70 19 32 40 33 12 67 41 29 52 60 42 21 24 3 6 59 5 62 36 17 61 44 31 4 57 10 45 39 63 51 14 37 7 8 53 38 11 9 58 34 25 22 35 49 20 69 23 48 46 56 30 2 68 54 43 13 55 18 27 15 26 71 64 66 16 47 -1
32 66 60 18 51 11 33 55 21 58 22 35 9 50 15 43 63 13 3 -1
20 5 43 60 51 37 48 34 24 4 70 53 30 38 61 54 56 68 50 49 40 33 39 16 12 21 59 27 69 66 35 22 71 11 45 52 31 -1
38 69 53 56 40 71 35 11 20 51 17 47 19 67 2 21 28 29 6 43 61 23 41 48 54 14 4 66 15 13 64 70 34 22 31 12 39 -1
23 37 71 22 -1
37 45 3 26 58 70 53 18 43 60 7 47 21 2 6 64 39 38 32 30 -1
11 21 10 16 23 20 22 37 38 25 59 34 67 43 5 61 -1
64 9 22 10 57 39 67 69 54 8 3 40 23 18 34 30 2 58 29 53 24 56 28 26 48 68 52 5 6 46 16 63 7 51 17 42 41 61 36 11 49 62 65 47 38 27 31 33 21 71 19 15 25 60 20 55 70 50 -1
8 29 12 40 4 47 38 28 49 3 33 11 30 52 6 46 5 71 20 68 70 22 34 60 43 54 35 9 45 67 27 -1
42 -1
42 46 3 60 38 57 31 47 4 70 15 64 43 44 17 11 13 33 41 40 62 53 56 29 63 66 -1
11 47 17 42 28 57 70 18 22 34 33 66 56 16 31 58 15 26 55 61 60 29 44 21 53 37 67 27 25 4 39 9 30 52 48 13 71 49 38 41 43 3 19 7 12 69 54 2 65 8 24 -1
52 39 46 51 19 5 25 16 9 28 40 67 42 54 37 11 66 22 63 56 41 36 68 26 24 29 7 49 18 -1
10 15 47 52 5 62 20 21 51 43 9 59 53 65 32 29 68 39 48 64 30 28 42 63 37 50 22 27 70 61 38 16 33 7 3 6 66 31 12 44 35 36 17 24 55 57 13 23 60 49 8 26 69 67 34 56 19 -1
59 21 38 37 53 39 63 50 56 62 10 69 30 55 24 68 61 4 7 20 44 54 17 25 14 43 34 13 22 46 6 57 47 71 19 29 35 26 12 27 28 52 65 23 9 3 32 42 15 5 58 -1
41 23 11 71 53 69 24 49 28 32 2 57 61 63 43 47 66 22 70 34 51 20 55 18 21 40 14 27 44 13 6 42 8 7 62 16 17 31 65 68 67 26 29 30 38 36 46 56 60 54 64 50 45 33 -1
52 15 40 18 19 21 7 66 5 50 59 39 63 36 71 58 32 44 68 35 11 3 62 57 34 38 64 54 12 20 25 16 60 43 30 2 23 22 67 27 46 33 -1
29 19 51 70 41 56 59 68 26 55 65 62 53 5 45 43 52 28 -1
6 8 23 47 54 17 52 71 57 41 49 37 -1
24 31 13 20 63 35 30 69 67 10 23 19 41 2 60 70 33 7 28 29 46 61 12 9 59 44 38 14 40 32 62 65 25 17 37 5 6 54 53 8 42 52 57 27 47 26 49 56 55 16 48 21 66 11 64 58 22 36 15 50 3 68 34 39 4 51 -1
65 54 5 51 47 44 23 66 18 27 3 67 41 30 13 2 11 8 9 39 33 42 50 29 38 25 70 57 37 35 28 71 17 62 58 52 26 59 56 60 10 64 34 24 45 68 61 22 21 6 36 40 7 53 46 31 43 19 48 4 49 32 -1
66 54 58 44 30 -1
51 44 6 37 69 24 50 52 26 64 27 65 18 2 49 22 58 70 47 13 20 8 33 66 3 45 71 17 38 -1
3 13 48 4 40 18 49 39 61 52 -1
47 2 20 63 66 49 68 30 16 50 70 51 38 -1
18 46 47 68 13 29 61 21 7 6 25 69 14 16 2 10 53 51 3 45 44 17 42 65 5 30 52 49 15 57 40 60 39 34 66 56 26 43 28 71 12 4 58 63 62 41 11 32 50 55 64 36 31 67 23 24 9 35 -1
34 27 37 25 39 28 42 60 41 9 15 3 36 21 55 52 32 62 59 22 46 70 43 35 44 67 61 49 19 63 38 24 58 65 48 18 14 64 31 -1
66 43 4 26 8 60 7 39 15 49 33 37 11 41 51 56 20 57 16 28 52 64 29 10 13 62 42 30 36 24 54 68 59 34 44 70 69 48 47 -1
58 40 26 68 22 39 60 48 65 71 13 8 21 14 19 38 33 30 35 59 53 11 32 50 69 46 52 17 15 56 66 54 -1
2 5 58 42 70 43 18 63 33 59 48 40 16 56 49 22 15 36 66 61 -1
17 69 61 66 58 9 24 2 52 41 20 42 45 37 27 63 23 38 46 48 28 64 51 6 36 19 29 32 53 25 16 21 3 33 44 12 -1
16 40 54 59 -1
2 53 68 39 25 21 52 13 41 38 22 27 44 33 43 64 37 31 24 17 -1
9 63 52 12 21 50 62 58 60 13 10 24 37 33 68 42 38 64 36 71 51 56 19 54 47 26 16 6 22 27 4 18 69 8 20 49 40 57 43 41 23 15 3 66 59 67 39 44 11 70 31 55 17 45 48 29 53 46 5 -1
69 30 13 55 46 50 4 59 20 45 21 37 60 12 2 5 6 52 27 32 70 48 14 62 47 65 57 36 61 71 39 51 34 35 31 44 64 53 -1
40 69 -1
49 40 17 66 67 13 8 36 16 33 26 25 58 50 59 19 64 24 38 5 56 65 27 29 14 44 31 46 57 45 53 70 20 69 42 22 55 52 23 54 18 71 28 9 7 41 61 48 68 21 15 47 12 2 63 60 -1
55 48 10 29 58 30 46 35 21 7 61 41 52 51 69 37 2 34 45 32 20 -1
32 18 12 7 57 41 2 31 45 15 8 50 59 22 30 37 27 20 39 63 60 64 47 16 5 36 24 6 49 29 43 10 14 21 33 68 19 62 46 70 4 25 34 23 67 54 44 65 3 48 28 55 56 9 40 66 53 61 42 38 69 71 -1
42 20 64 48 30 63 -1
48 55 14 42 61 2 7 11 20 63 49 6 8 32 38 56 31 45 22 10 47 35 69 30 29 -1
13 40 66 32 63 4 -1
11 16 15 62 47 70 57 20 41 33 35 21 49 43 24 37 60 50 12 63 56 69 28 58 55 40 22 6 29 66 13 52 4 32 54 67 36 17 19 61 59 48 45 26 14 27 5 30 9 2 64 31 71 46 23 25 53 10 68 -1
63 7 31 53 35 71 69 49 15 64 61 68 57 23 29 47 55 56 6 38 54 10 70 17 28 3 19 25 65 30 2 8 9 51 11 32 5 21 59 24 27 48 18 43 50 67 26 22 16 33 4 34 45 58 40 42 -1
31 4 43 21 54 59 32 24 46 61 70 5 40 65 19 64 6 28 16 67 11 66 10 35 13 63 26 50 36 39 27 34 -1
19 67 38 66 16 59 31 43 24 51 8 13 9 46 64 62 5 -1
48 2 67 20 62 6 49 3 25 58 69 7 71 22 61 19 51 21 70 10 53 60 64 16 27 26 11 37 34 24 43 33 45 57 8 38 50 28 15 55 39 13 9 56 68 36 29 63 40 44 41 66 4 18 47 52 35 23 5 12 17 14 42 30 54 46 31 65 -1
31 50 32 3 60 33 11 41 34 20 26 62 54 13 71 12 6 4 61 10 29 15 16 42 17 56 65 67 69 23 27 48 64 22 44 25 68 7 8 52 40 55 14 5 30 43 38 70 49 39 21 53 59 28 51 18 47 66 63 -1
48 64 39 15 70 22 62 9 37 42 21 14 68 40 13 27 71 11 46 8 30 61 57 45 66 29 63 65 67 51 17 41 10 5 52 47 54 25 55 3 16 4 44 26 32 59 38 58 36 31 56 23 28 60 7 34 33 2 18 24 6 19 49 69 -1
49 11 10 47 34 36 33 62 32 63 44 19 2 27 5 25 46 15 37 45 68 8 28 26 9 3 50 65 6 71 67 66 54 56 24 17 35 30 14 18 12 52 39 48 43 60 53 -1
42 51 37 54 68 25 46 6 -1
35 64 57 6 37 70 8 5 69 33 42 59 31 21 22 13 44 67 58 16 47 39 53 63 40 60 66 11 43 41 34 14 68 23 9 17 48 38 10 49 27 55 65 20 51 3 12 54 15 18 61 52 28 62 45 7 26 30 4 2 36 71 56 19 46 32 50 24 25 29 -1
60 23 21 28 52 13 22 46 70 50 64 41 42 66 55 5 49 37 9 18 27 62 4 19 26 32 58 16 54 71 69 57 48 -1
65 26 27 34 59 24 31 9 50 32 44 11 39 61 23 12 6 62 46 43 68 10 21 71 40 51 54 57 56 3 45 5 63 28 70 60 58 35 33 17 7 14 47 25 22 19 29 64 69 42 55 18 53 41 49 -1
56 20 65 42 24 21 46 30 12 14 29 26 59 64 41 31 68 57 13 8 62 61 66 43 7 50 9 44 34 -1
