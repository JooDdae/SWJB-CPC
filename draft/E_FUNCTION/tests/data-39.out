YES
9 17 32 41 47 50 54 55 